IDGPredict/FacetPredict.h

```diff
@@ -81,34 +88,38 @@ class FacetPredict
     int buffersize = 256;
     idg::api::options_type options;
     IdgConfiguration::Read(proxyType, buffersize, options);
-    ao::uvector<double> data;
+    std::vector<ao::uvector<double>> data(_readers.size());
     _metaData.clear();
+    FitsReader& reader = _readers.front();
```

Comment:
If I understand correctly, you can now provide a fits file with spectral index (and possibly higher order terms). Maybe add an assert here that all sizes are equal? I think otherwise stuff will crash.

Reply:
Are you referring to whether the images have the same size? That's a good idea, I've now added checks here:
https://github.com/lofar-astron/DP3/blob/497395ff94bb9176bfca0063f69b048d3fc3ddc6/IDGPredict/FacetPredict.h#L44
The number of spectral terms is determined from the number of images in the parset.